src/main/kotlin/de/danielmaile/mpp/MPP.kt

```diff
@@ -268,3 +277,11 @@ fun inst(): MPP {
 fun aetherWorld(): World {
     return MPP.instance.worldManager.aetherWorld
 }
+
+// FIXME: I don't really like this here it's quite unintuitive, but I need it for the resource pack builder
```

Comment:
We should either fix it or remove the comment.

Reply:
How do you suggest to fix it?
That's the point of the comment.

Reply:
I would just leave it as is. I'm not aware of any "fix" for this problem.